k8s/tools/deploy-celery-gke.sh

```diff
@@ -0,0 +1,245 @@
+# Requirements:
+# - have 'gcloud' and 'kubectl' installed.
+# - autheticate against your GCP project with "gcloud auth login"
+# - account being used to run script should have an IAM policy of instance.admin and container.admin used to create the necessary resources.
+# - optionally have the GCP project set with "gcloud config set project [you-project-name]"
+#
+# Use --help to show you commands supported.
+
+set -o posix
+set -e
+
+# Source cluster config to pull specs to create cluster from. Please review
+# the config file and make any necessary changes prior to executing this script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source $DIR/.clusterconfig
+cd $DIR/..
+
+if [[ "$*" == *--help ]] ; then
+  echo "Turbinia deployment script for Kubernetes environment"
+  echo "Options:"
+  echo "--build-dev                    Deploy Turbinia development docker image"
+  echo "--build-experimental           Deploy Turbinia experimental docker image"
+  echo "--no-cluster                   Do not create the cluster"
+  echo "--no-filestore                 Do not deploy Turbinia Filestore"
+  echo "--deploy-dfdewey               Deploy dfDewey datastores"
+  exit 1
+fi
+
+# Check if gcloud is installed
+if [[ -z "$( which gcloud )" ]] ; then
+  echo "gcloud CLI not found.  Please follow the instructions at "
+  echo "https://cloud.google.com/sdk/docs/install to install the gcloud "
+  echo "package first."
+  exit 1
+fi
+
+# Check if kubectl is installed
+if [[ -z "$( which kubectl )" ]] ; then
+  echo "kubectl CLI not found.  Please follow the instructions at "
+  echo "https://kubernetes.io/docs/tasks/tools/ to install the kubectl "
+  echo "package first."
+  exit 1
+fi
+
+# Check configured gcloud project
+if [[ -z "$DEVSHELL_PROJECT_ID" ]] ; then
+  DEVSHELL_PROJECT_ID=$(gcloud config get-value project)
+  ERRMSG="ERROR: Could not get configured project. Please either restart "
+  ERRMSG+="Google Cloudshell, or set configured project with "
+  ERRMSG+="'gcloud config set project PROJECT' when running outside of Cloudshell."
+  if [[ -z "$DEVSHELL_PROJECT_ID" ]] ; then
+    echo $ERRMSG
+    exit 1
+  fi
+  echo "Environment variable \$DEVSHELL_PROJECT_ID was not set at start time "
+  echo "so attempting to get project config from gcloud config."
+  echo -n "Do you want to use $DEVSHELL_PROJECT_ID as the target project? (y / n) > "
+  read response
+  if [[ $response != "y" && $response != "Y" ]] ; then
+    echo $ERRMSG
+    exit 1
+  fi
+fi
+
+# TODO: Do real check to make sure credentials have adequate roles
+if [[ $( gcloud -q --project $DEVSHELL_PROJECT_ID auth list --filter="status:ACTIVE" --format="value(account)" | wc -l ) -eq 0 ]] ; then
+  echo "No gcloud credentials found.  Use 'gcloud auth login' and 'gcloud auth application-default login' to log in"
+  exit 1
+fi
+
+# Enable IAM services
+gcloud -q --project $DEVSHELL_PROJECT_ID services enable iam.googleapis.com
+
+# Create Turbinia service account.
```

Comment:
Can you add a comment about what this SA is used for?  i.e. AFAIK this is the SA key that Turbinia will actually use at run time (I just wanted to differentiate between the SA key we used to create that the client would use).